subprojects/gst-docs/examples/tutorials/meson.build

```
@@ -0,0 +1,37 @@
+project('tutorials', 'c')
+
+dependencies = []
+dependencies += dependency('gstreamer-1.0')
+dependencies + = dependency('gtk+-3.0', version : '>=3.0')
+dependencies + = dependency('clutter-gst-3.0', version : '>=3.0')
+dependencies + = dependency('gstreamer-audio-1.0')
+dependencies + = dependency('gstreamer-pbutils-1.0')
```

Meson fails to configure. There is an extra space between `+` and `=`.

```suggestion
dependencies += dependency('gstreamer-1.0')
dependencies += dependency('gtk+-3.0', version : '>=3.0')
dependencies += dependency('clutter-gst-3.0', version : '>=3.0')
dependencies += dependency('gstreamer-audio-1.0')
dependencies += dependency('gstreamer-pbutils-1.0')
```

---

Done